tests/integration/mirrorpeer_status_test.go

```diff
@@ -0,0 +1,191 @@
+package integration_test
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	multiclusterv1alpha1 "github.com/red-hat-storage/odf-multicluster-orchestrator/api/v1alpha1"
+	"github.com/red-hat-storage/odf-multicluster-orchestrator/controllers"
+	"github.com/red-hat-storage/odf-multicluster-orchestrator/controllers/common"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	addonapiv1alpha1 "open-cluster-management.io/api/addon/v1alpha1"
+	clusterv1 "open-cluster-management.io/api/cluster/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	fakeMirrorPeerName = "fake-mirror-peer"
+	testCluster1       = "test-provider-cluster1"
+	testCluster2       = "test-provider-cluster2"
+	testSC1            = "test-storagecluster-1"
+	testSC2            = "test-storagecluster-2"
+	testSCNamespace    = "test-namespace"
+)
+
+func GetFakeMirrorPeerReconciler() (*controllers.MirrorPeerReconciler, *multiclusterv1alpha1.MirrorPeer, error) {
```

Comment:
A reconciler is already created here: https://github.com/red-hat-storage/odf-multicluster-orchestrator/blob/main/tests/integration/suite_test.go#L96-L100 and runs for all tests. We don't need to create it separately.